bokehjs/src/lib/models/tools/inspectors/hover_tool.ts

```diff
@@ -376,11 +384,14 @@ export class HoverToolView extends InspectToolView {
 
   _emit_callback(geometry: PointGeometry | SpanGeometry): void {
     for (const r of this.computed_renderers) {
+      if (!(r instanceof GlyphRenderer))
+        continue
+
       const rv = this.plot_view.renderer_views.get(r)!
       const x = rv.coordinates.x_scale.invert(geometry.sx)
       const y = rv.coordinates.y_scale.invert(geometry.sy)
 
-      const index = (r as any).data_source.inspected
+      const index = r.data_source.inspected as any // XXX: Selection vs number
```

Comment:
@bryevdv, do you know what was the intent here? Either types are wrong or this code doesn't make sense.

Reply:
I don't at all, my guess is this is a holdover of some change that happened then the proper `Selection` model was introduced.